src/main-process/modules/dock-menu.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Module in charge of the dock menu on macOS
+ */
+
+const { Menu, app, ipcMain } = require('electron');
+
+const ModuleWindow = require('./module-window');
+const { IPCR_PLAYER_ACTION } = require('../../shared/constants/ipc');
+
+
+class TrayManager extends ModuleWindow {
+  constructor(window) {
+    super(window);
+    this.platforms = ['darwin'];
+  }
+
+  load() {
+    this.songDetails = [
+      {
+        label: 'Not playing',
+        enabled: false,
+      },
+      {
+        type: 'separator',
+      },
+    ];
+
+    this.playToggle = [
+      {
+        label: 'Play',
+        click: () => {
+          this.window.webContents.send(IPCR_PLAYER_ACTION, 'play');
+        },
+      },
+    ];
+
+    this.pauseToggle = [
+      {
+        label: 'Pause',
+        click: () => {
```

Comment:
can't we use short methods notation from ES6?
```js
{
  label: 'Pause',
  click() {
    this.window.webContents.send(IPCR_PLAYER_ACTION, 'pause');
  },
},
```

Reply:
Because the context is different, and `this` won't refer to the module itself but to a global window object.